### PR TITLE
grammar

### PR DESCRIPTION
**Pull request by barak**

### CODE REVIEW

File: src/mlpack/bindings/cli/third_party/CLI/CLI11.hpp [hunk 1]
```diff
@@ -691,7 +691,7 @@ class CallForVersion : public Success {
         : CallForVersion("This should be caught in your main function, see examples", ExitCodes::Success) {}
 };
 
-/// Does not output a diagnostic in CLI11_PARSE, but allows to return from main() with a specific error code.
+/// Does not output a diagnostic in CLI11_PARSE, but allows main() to return with a specific error code.
```
Comment (zoq) [inline src/mlpack/bindings/cli/third_party/CLI/CLI11.hpp:694]: I guess since this is a third party header fill, we should fix this upstream.

**Review by zoq — COMMENTED** [review]
Nice, just a minor comment about the fix in the `CLI11.hpp` file.

**Review by birm — APPROVED** [review]

File: src/mlpack/bindings/cli/third_party/CLI/CLI11.hpp (hunk 1 above)
Comment (rcurtin) [reply src/mlpack/bindings/cli/third_party/CLI/CLI11.hpp:694]: https://github.com/CLIUtils/CLI11/pull/510

**Comment by rcurtin** [comment]
It looks like the builds are failing because of unrelated problems like #2601, so I'll just go ahead and merge this since it really shouldn't be able to break any builds at all (it's just wording fixes).  Thanks @barak! :+1:

**Pull request merged**